actionview/lib/action_view/railtie.rb

```diff
@@ -39,7 +39,7 @@ class Railtie < Rails::Engine # :nodoc:
 
     initializer "action_view.per_request_digest_cache" do |app|
       ActiveSupport.on_load(:action_view) do
-        if app.config.consider_all_requests_local
+        if app.config.consider_all_requests_local && !Rails.env.test?
           app.executor.to_run ActionView::Digestor::PerExecutionDigestCacheExpiry
```

Comment:
Just to clarify. Does reloader run when templates changed, or it tracks only .rb files?